indico/legacy/webinterface/tpls/RoomBookingNewBookingPeriodWidget.tpl

```diff
@@ -61,6 +61,15 @@
                 </div>
             </div>
         % endif
+        <div class="error-message-box booking-limit-warning hidden">
+            <div class="message-text">
+                ${_('It is not possible to book the selected rooms for more than {0} days'.format(booking_limit)) }
```

Comment:
So I guess [this](https://github.com/indico/indico/pull/2884/files/2882dc9f620942f54711a18280892b1c0824f243#diff-8d79621315f52d4f1c51c2bd1e03cb44R42) is wrong as well.